constants.py

```diff
@@ -0,0 +1,3 @@
+MAIN_URI = "main/"
```

Comment:
Move this file inside "main" folder ... That is because only this particular application is dealing with this constants file

Reply:
Actually split the file in two constants file. One constants you can keep as it is. Create another constants file inside "main" folder ... Then inside this file, keep routes and other constants which are relevant only to "main".

we have a constants file "./constants.py" which is a constants file throughout the CabAd project. Keep MAIN_URI and ADMIN_URI constants here.

create another constants file in "./main/constants/urls.py" and keep like TEAM_URI and other URIs there